pkg/libfixmath/Makefile.dep

```diff
@@ -6,3 +6,9 @@ ifneq (,$(filter libfixmath-unittests,$(USEMODULE)))
   # The round function is not provided by the msp430 toolchain
   FEATURES_BLACKLIST += arch_msp430
 endif
+
+ifneq (,$(filter arm%,$(CPU_ARCH)))
```

Comment:
This would mean a much bigger change, it would mean that `TOOLCHAIN` has dependencies, and since the default `TOOLCHAIN` depends on the `OS`... not sure how to deal with this

Reply:
Not necessarily. If a different TOOLCHAIN is set from the command line, at this stage you know it already. And gcc the default toolchain anyway.

Some something like the following would work:

```mk
ifneq (llvm,$(TOOLCHAIN))
  FEATURES_BLACKLIST += arch_cortexm
endif
```

I know it's a bit hacky but that the minimal change to skip the build on arm with gcc but not with llvm.